services/distribution/src/main/resources/application.yaml

```diff
@@ -113,8 +113,8 @@ services:
       bucket: ${STATISTICS_FILE_S3_BUCKET:obs-cwa-public-dev}
 
     app-features:
-      - label: reserved
-        value: ${reserved:1}
+      - label: unencrypted-checkins-enabled
+        value: ${EVREG_UNENCRYPTED_CHECKINS_ENABLED:0}
```

Comment:
I think boolean (true/false) is easier to understand than 1/0 as is done in the submission yaml. Also decide for one.

Reply:
@hilmarf  why do we have that? First time I see this this config and it is not used inside distribution.

Reply:
it's the mobile app-feature toggle - becomes part of the mobile app-config, which is created by the distribution

Reply:
ok so that means there is nothing to do here right?

Reply:
yes